e2e_deployment_files/long_haul_deployment.template.json

```diff
@@ -435,6 +438,12 @@
           "timeToLiveSecs": 86400
         }
       }
+    },
+    "metricsCollector": {
+      "properties.desired": {
+        "testType": "longhaul",
+        "testId": "<TrackingId>"
```

Comment:
add upstream protocol?

Reply:
Good idea

Reply:
Longhaul only uses amqp as upstream so I will hardcode this